src/styles/_buttons.scss

```diff
@@ -75,10 +75,18 @@
 
 .btn-language-student {
   @include button-variant($color-green-dark, transparent, $color-green-dark);
+  &:focus {
+    border-style: solid;
```

Comment:
Alle de "vanlige" outlinene (i chrome i hvertfall) er tykkere... skulle vi laget disse mer lik? Eller er tanken at alle outlines skal bli tynnere etterhvert?

Reply:
Det som kommer i focus til vanlig når man tabber rundt på siden er samme størrelse. Det er bare akkurat den lastLesson-knappen som blir dobbelt opp med focus siden det ikke er en dropdown som de to andre

Reply:
Dvs den lastLesson-knappen er sånn det egentlig burde være, men dropdown-knappene i navbar er litt for mye hacket til